openwisp_controller/config/api/serializers.py

```diff
@@ -41,3 +47,51 @@ class ListOrgTemplateSerializer(ListTemplateSerializer):
     openwisp-controller list template serializer
     """
     pass
+
+
+class ListCreateTemplateSerializer(TemplateDetailSerializer, serializers.Field):
```

Comment:
you may call this just `TemplateSerializer`, unless there are more template serializers